package.json

```diff
@@ -28,7 +28,7 @@
     "partial-lint": "git diff --name-only master... | grep js$ | xargs ./node_modules/.bin/eslint",
     "check": "npm run lint && npm run test",
     "tempcheck": "npm run partial-lint && npm run test",
-    "test": "jest --watch"
```

Comment:
Was going to say this probably wouldn't work well with the CI tools. Most of the npm scripts are designed for those tools. If you want to add more for use during local development then please do add them

Reply:
@lukekearney Yep, totally didn't mean to commit it! :whale: